testplan/runnable/base.py

```diff
@@ -228,7 +233,7 @@ class TestRunnerResult(RunnableResult):
     def __init__(self):
         super(TestRunnerResult, self).__init__()
         self.test_results = OrderedDict()
-        self.exporter_results = []
+        self.export_context = ExportContext()
```

Comment:
I'm not sure the whole context is needed for the results. The context is for the exporters to communicate, and each generating results, so keep it here